modules/avatar/react/stories/stories_avatarButton.tsx

```diff
@@ -0,0 +1,60 @@
+/// <reference path="../../../../typings.d.ts" />
+import * as React from 'react';
+import {storiesOf} from '@storybook/react';
+import withReadme from 'storybook-readme/with-readme';
+
+import {AvatarButton} from '../index';
+import README from '../README.md';
+
+const IMAGE_URL = 'https://s3-us-west-2.amazonaws.com/design-assets-internal/avatars/lmcneil.png';
+
+storiesOf('AvatarButton', module)
+  .addDecorator(withReadme(README))
+  .add('Light', () => (
+    <div className="story">
+      <h3>Extra-Extra Large</h3>
+      <AvatarButton size={AvatarButton.Size.xxl} />
```

Comment:
Do you mind adding a mock `onClick` to each of these so they're not disabled?

Reply:
sure